src/EditorFeatures/VisualBasic/BraceMatching/StringLiteralBraceMatcher.vb

```diff
@@ -16,14 +15,15 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.BraceMatching
             Dim root = Await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(False)
             Dim token = root.FindToken(position)
 
-            If token.Kind = SyntaxKind.StringLiteralToken AndAlso Not token.ContainsDiagnostics Then
-                Return New BraceMatchingResult(
-                    New TextSpan(token.SpanStart, 1),
-                    New TextSpan(token.Span.End - 1, 1))
+            If position = token.SpanStart OrElse position = token.Span.End - 1 Then
```

Comment:
this change was made so that if you start navigating around *inside* a string, we don't return matching brace results here (which would then cause it to *not* call into the embedded language brace matcher).  Note that we *always* returned brace matching results for strings (even when in the middle of them).  But we then filtered these out because the caret wasn't actually touching the braces.